spyder/plugins/outlineexplorer/widgets.py

```diff
@@ -29,6 +29,8 @@
                                     create_waitspinner)
 from spyder.widgets.onecolumntree import OneColumnTree
 
+CELL = 225
+BLOCK_COMMENT = 224

```

Comment:
- Please add blank lines above and below these two.
- Also move `BLOCK_COMMENT = 224` to be above `CELL = 225` because it's associated number is smaller.
- Finally, add a comment on top of them like this

      # Additional symbol constants